barman/retention_policies.py

```diff
@@ -126,6 +126,29 @@ def to_json(self):
         """
         return "%s %s %s" % (self.mode, self.value, self.unit)
 
+    def propagate_retention_status_to_children(self, backup_info, report, ret_status):
+        """
+        Propagate retention status to all backups in the chain.
+
+        .. note::
+            This has a side-effect. It modifies or add data to *report* dict.
+
+        :param barman.infofile.BackupInfo backup_info: The object we want to
+            propagate the RETENTION STATUS from.
+        :param report dict[str, str]: The report data structure to be modified.
```

Comment:
```suggestion
        :param dict[str, str] report: The report data structure to be modified.
```

Type comes before the name.